Equilobe.DailyReport.Models/Storage/AtlassianWorklog.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace Equilobe.DailyReport.Models.Storage
+{
+    public class AtlassianWorklog
+    {
+        public long Id { get; set; }
+        public long InstalledInstanceId { get; set; }
+        public long AtlassianUserId { get; set; }
+        public long IssueId { get; set; }
+        public string IssueKey { get; set; }
+        public long JiraWorklogId { get; set; }
```

Comment:
What does this represent? The id in JIRA?

Reply:
Yeah. I have the id which is in our database for our worklog and the id which is in Jira. I decided to have this id as well in order to make searches if there were any updates.